packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -276,7 +276,9 @@ export class Thread implements IUniqueId {
       reactionType,
       addressesReacted
     );
-    this.latestActivity = latest_activity ? moment(latest_activity) : null;
```

Comment:
Can we remove this `latest_activity` argument entirely? Or does it still serve a purpose?

Reply:
There might be some client-side sorting involved, particularly with the `sortByFeaturedFilter`. 
<img width="1129" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/4791635/7c2607b5-bac6-40f3-bf61-54f76411ae1e">

Though my understanding is client side only sorting is not feasible due to pagination all requests are sent to backend
Can anyone on the frontend team confirm this? 

I've kept `Thread` model same as before.
Only slight change in how we derive `latest_activity`.

Reply:
My point is that we ignore the `latest_activity` arg provided to the constructor now -- and I want to be clear that this is OK and also whether it has upstream implications.

Reply:
`  const threads = sortPinned(sortByFeaturedFilter(data || [], featuredFilter));`

This is where it is used on frontend - may be somebody from frontend can confirm and remove.
We might not need many other columns and helper function `sortByFeaturedFilter` if not needed.

Reply:
I think you're missing my point here, which is we have an _unused variable warning_ now on the client, since we're using `last_commented_on` to set `this.latestActivity` rather than using `latest_activity` like we previously were. So does that mean we can remove the `latest_activity` query altogether? + we should definitely remove the `latest_activity` argument to the Thread constructor.

Reply:
ohh i see the confusion missed reading the whole expression due to formatting:
```
this.latestActivity = last_commented_on
      ? moment(last_commented_on)
      : moment(created_at);
```

There can be `Threads` without any comments
Its equivalent of earlier expression on backend `COALESCE(t.last_commented_on, t.created_at)`

Reply:
> we should definitely remove the latest_activity argument to the Thread constructor.

Yeah let me remove this

Reply:
Can we remove that field latest_activity from the query as well, since the constructor doesn't use it? Or is it used elsewhere in the codebase?